pnpm-workspaces/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@bazel-poc/main",
+  "version": "0.0.0",
+  "devDependencies": {
+    "@tsconfig/node16-strictest": "^1.0.4",
+    "@types/node": "^18.8.5",
+    "typescript": "^4.8.4"
+  },
+  "dependencies": {
+    "inspirational-quotes": "^2.0.1"
+  },
+  "scripts": {
+    "build": "bazelisk build ...",
```

Comment:
Do you normally launch bazel with pnpm (or yarn, or npm, ...)? I think we can leave these out?

Reply:
I don't, but devs do. But I can remove them for the example. 👍

Reply:
Just to keep the example minimal 👍